tiled/client/node.py

```diff
@@ -543,6 +549,87 @@ def _ipython_key_completions_(self):
             # Do not print messy traceback from thread. Just fail silently.
             return []
 
+    def create_array(self, array, metadata=None, specs=None):
+        """
+        EXPERIMENTAL: write an array
+
+        This is subject to change or removal without notice
+
+        """
+
+        from ..structures.array import ArrayMacroStructure, ArrayStructure, BuiltinDtype
+
+        metadata = metadata or {}
+        specs = specs or []
+        mimetype = "application/octet-stream"
+
+        structure = ArrayStructure(
+            macro=ArrayMacroStructure(
+                shape=array.shape,
+                # just one chunk for now...
+                chunks=tuple((size,) for size in array.shape),
+            ),
+            micro=BuiltinDtype.from_numpy_dtype(array.dtype),
+        )
+        data = {
+            "metadata": metadata,
+            "structure": asdict(structure),
+            "structure_family": StructureFamily.array,
+            "specs": specs,
+            "mimetype": mimetype,
+        }
+        document = self.context.post_json("/node/metadata/", data)
```

Comment:
If we write to a node like `/analyzed` this should go to `/node/metadata/analyzed/{uid}`. This example is relevant:

https://github.com/bluesky/tiled/blob/16478d9e55b6f1f401b98d9fe042e3d0ed4d7f80/tiled/client/array.py#L49-L53